pkgs/build-support/substitute/substitute.sh

```diff
@@ -8,7 +8,13 @@ if test -n "$dir"; then
     mkdir -p $out/$dir
 fi
 
-substitute $src $target $replacements
+substitutionsList=($replacements)
```

Comment:
Perhaps add a little TODO to remove this once replacements is done away with?

Reply:
I'll add a TODO in the Nix file at the warning site :)